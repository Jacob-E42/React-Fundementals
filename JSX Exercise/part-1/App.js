const App = () => {
	return (
		<div>
			<FirstComponent />
			<NamedComponent name="myself" />
		</div>
	);
};
