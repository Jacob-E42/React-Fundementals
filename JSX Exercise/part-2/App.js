const App = () => {
	return (
		<div>
			<Tweet username="jello" name="Damian Green" Date="3/10/23" message="HI!!" />
			<Tweet username="tweetMaster66" name="Horse Face" Date="3/10/23" message="I'm the best" />
			<Tweet username="gelatin" name="Copy Cat" Date="3/10/23" message="HI!!" />
		</div>
	);
};
