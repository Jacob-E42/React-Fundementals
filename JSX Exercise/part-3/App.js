const App = () => {
	return (
		<div>
			<Person name="Johnathan" age={29} hobbies={["kites", "making model airplanes", "skiing"]} />
			<Person name="Samantha" age={63} hobbies={["bungee jumping", "skydiving", "jetskiing"]} />
			<Person name="Max" age={17} hobbies={["tic tac toe", "chess", "video games"]} />
		</div>
	);
};
