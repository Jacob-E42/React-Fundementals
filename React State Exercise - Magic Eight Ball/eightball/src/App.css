.App {
	text-align: center;
	display: flex;
	flex-direction: column;
}

.App-header {
	background-color: #6f727a;
	width: 50vw;
	margin: 2vh;
	padding: 1vh;
	display: flex;
	flex-direction: column;
	align-items: center;
	align-self: center;
	justify-content: center;
	font-size: calc(10px + 2vmin);
	color: white;
}
