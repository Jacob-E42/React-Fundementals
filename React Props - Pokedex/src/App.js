import React from "react";
import "./App.css";
import Pokedex from "./Pokedex";

function App(props) {
	return (
		<div className="App">
			<Pokedex pokemon={props.pokemon} />
		</div>
	);
}

export default App;
