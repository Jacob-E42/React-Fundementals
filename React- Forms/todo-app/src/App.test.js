import { render, screen } from "@testing-library/react";
import App from "./App";

test("smoketestand snapshot test  for App", () => {
	const { asFragment } = render(<App />);
	expect(asFragment).toMatchSnapshot();
});
