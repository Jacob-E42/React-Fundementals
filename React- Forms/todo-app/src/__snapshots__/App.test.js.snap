// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoketestand snapshot test  for App 1`] = `[Function]`;
